### PR TITLE
feature/file naming

### PR DESCRIPTION
**Pull request by sithumonline**
- Fix css build issue
- Add husky and prettier
- Add prettier config
- Add as dev deps
- Update eslintrc
- Add plugin
- Add tslint
- Fix src errors
- Rename files and eslint fix

### CODE REVIEW

**Comment by sithumonline** [comment]
@LayanJay follows current file naming conventions.

File: modules/HomePage/OurPartners.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          <div className="flex items-center justify-center w-9 sm:w-16 md:w-20 lg:w-28 my-8">
+            <Image
+              src={fcsc}
+              alt="Faculty of computing Student Community"
+              layout="intrinsic"
+              quality={90}
+              placeholder="blur"
+            />
+          </div>
+
```
Comment (chamodshehanka) [inline modules/HomePage/OurPartners.tsx:45]: These Images are repetitive, load them using a `map()`

**Review by chamodshehanka — CHANGES_REQUESTED** [review]

**Comment by LayanJay** [comment]
oops. I've also fixed the file names.  Should I want to discard my changes? as it may lead to merging conflicts

File: modules/HomePage/OurPartners.tsx (hunk 1 above)
Comment (LayanJay) [reply modules/HomePage/OurPartners.tsx:45]: I'll fix this

**Pull request closed without merging**